assets.src/src/redmine-tracky/controllers/timer-controller.ts

```diff
@@ -56,18 +55,28 @@ export default class extends Controller {
         }
     }
 
-    private timeDifference(): string {
-        const startDateTime = this.convertToDateTime((this.startTarget as HTMLInputElement).value);
-        const endDateTime = this.convertToDateTime((this.endTarget as HTMLInputElement).value);
+    private timeDiffToString(timeDiff: TimeDiff) {
+        const sign = (timeDiff.minutes < 0 || timeDiff.seconds < 0) ? '-' : '';
+    
+        return sign + ['hours', 'minutes', 'seconds']
+            .map((v) => timeDiff[v as keyof TimeDiff])
+            .map((v) => Math.abs(Math.floor(v)))
```

Comment:
did do conciously decide for floor instead of round?

Reply:
@sihu i changed it because it would round to e.g. `10:60` before resetting to `11:00`